parquet-column/src/main/java/org/apache/parquet/column/ParquetProperties.java

```diff
@@ -92,11 +93,12 @@ public static WriterVersion fromString(String name) {
   private final int statisticsTruncateLength;
   private final int pageRowCountLimit;
   private final boolean pageWriteChecksumEnabled;
+  private final boolean enableByteStreamSplit;
 
   private ParquetProperties(WriterVersion writerVersion, int pageSize, int dictPageSize, boolean enableDict, int minRowCountForPageSizeCheck,
                             int maxRowCountForPageSizeCheck, boolean estimateNextSizeCheck, ByteBufferAllocator allocator,
                             ValuesWriterFactory writerFactory, int columnIndexMinMaxTruncateLength, int pageRowCountLimit,
-                            boolean pageWriteChecksumEnabled, int statisticsTruncateLength) {
+                            boolean pageWriteChecksumEnabled, int statisticsTruncateLength, boolean enableByteStreamSplit) {
```

Comment:
While we keep adding, the parameter list becomes so long. It is not just because of the one you added, so it is OK for this change. But I think we need to refactor it later. I created PARQUET-1764 to address it.